### PR TITLE
Add backend throttle duration

### PR DESCRIPTION
**Pull request by EmileVauge**
This PR adds a backend throttle duration. It resolves https://github.com/EmileVauge/traefik/issues/46
Backends throttle duration is the minimum duration between 2 events from providers before applying a new configuration. It avoids unnecessary reloads if multiples events are sent in a short amount of time.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -107,6 +107,16 @@ For example:
...
 # KeyFile = "traefik.key"
+
+# Backends throttle duration: minimum duration between 2 events from providers
+# before applying a new configuration. It avoids unnecessary reloads if multiples events
+# are sent in a short amount of time.
+#
+# Optional
+# Default: "2s"
+#
+# BackendsThrottleDuration = "5s"
```
Comment (EmileVauge) [inline docs/index.md]: Maybe the name should be changed... Backend can be mingled with load-balancing backends... `ProvidersThrottleDuration` ?

File: integration/marathon_test.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func (s *MarathonSuite) TestSimpleConfiguration(c *check.C) {
-	cmd := exec.Command(traefikBinary, "fixtures/consul/simple.toml")
+	cmd := exec.Command(traefikBinary, "fixtures/marathon/simple.toml")
```
Comment (vdemeester) [inline integration/marathon_test.go]: Nice catch :sweat_smile:

File: docs/index.md (hunk 1 above)
Comment (vdemeester) [inline docs/index.md]: :+1:

**Comment by vdemeester** [comment]
LGTM and carrying it because I took too much time and so many `merge`/`rebase` to do :wink:

**Pull request closed without merging**